LibNoDaveConnectionLibrary/DataTypes/Projectfolders/Step7V5/BlocksOfflineFolder.cs

```diff
@@ -343,6 +340,14 @@ private tmpBlock GetBlockBytes(ProjectBlockInfo blkInfo)
                                 // I do not know what value for SFB
                                 myTmpBlk.IsInstanceDB = true;
                                 myTmpBlk.FBNumber = (int)ssbpart[1] + 256 * (int)ssbpart[2];
+
+                                //if this is an interface DB, then rather take the Interface declaration from the instance FB,
+                                //instead of the data sotred in the project. 
+                                //The reason is that if you change the comment in an FB, the DB data is not actualized and my contain outdated
+                                //Declarations. When you change the interface, Step7 tells you to "regenerate" the instance DB which only then would 
+                                //Actualize the comments. Simple Commentary changes do not change the Datablocks row. 
+                                tmpBlock InstFB = GetBlockBytes("FB" + myTmpBlk.FBNumber);

```

Comment:
Read the Declaration from the Function block, and overwrite the one read from the S7 Project files